pkg/util/log/logcrash/crash_reporting.go

```diff
@@ -255,6 +255,7 @@ func SetupCrashReporter(ctx context.Context, cmd string) {
 	crashReportingActive = true
 
 	sentry.ConfigureScope(func(scope *sentry.Scope) {
+		scope.SetTags(getTagsFromEnvironment())
```

Comment:
My worry is that the second SetTags call would override what we have here. Are we certain that it appends the tags rather than setting them? If that's not the case, we may want to merge with the map below.

Reply:
Update: looks like we're good here :smile: https://github.com/getsentry/sentry-go/blob/8b3833f0e2586da9a6a7b97e2b8e5f71e5846baf/scope.go#L209-L217

Reply:
We actually call SetTags in several places with different tags, so I'm not worried about this. Plus, here's the implementation:
```go
// SetTags assigns multiple tags to the current scope.
func (scope *Scope) SetTags(tags map[string]string) {
	scope.mu.Lock()
	defer scope.mu.Unlock()

	for k, v := range tags {
		scope.tags[k] = v
	}
}
```